update icon: phoenix (original-wordmark, plain-wordmark)

cyberalien opened the pull request:
## Double check these details before you open a PR

<!-- Tick the checkboxes to ensure you've done everything correctly -->
- [x] PR does not match another non-stale PR currently opened

## Features
Fixes `phoenix-*-wordmark` icons

**This PR closes 1528**
https://github.com/devicons/devicon/issues/1528

## Notes
Removes raster image in `original` icon.
Removes gradients in `plain` icon.

lunatic-fox commented:
Hello there!
First of all, thank you so much for verifying Devicon's icons. That's awesome! 🤩
Some icons were outdated that's why I prefer to make another PR.
Let me know if it's ok. 🙂

cyberalien commented:
Sure. I've just modified existing icon for this one, didn't check if it can be updated.

Updated icons are always better :) Closing this in favour of your PR.
